spec/SpecHelper.js

```diff
@@ -0,0 +1,15 @@
+beforeEach(function () {
```

Comment:
You can probably remove this code and replace it with other code you might need to define in order to mock the Frame class 🙂